xwiki-pro-macros-test/xwiki-pro-macros-test-pageobjects/src/main/java/com/xwiki/pro/test/po/generic/TeamMacroPage.java

```diff
@@ -37,11 +38,82 @@ public class TeamMacroPage extends ViewPage
     @FindBy(css = ".xwikiteam")
     private List<WebElement> teamMacros;
 
```

Comment:
Same for this class, I would use a separate TeamMacro (along with the Page, or just alone, depending on the code)